src/sbt-test/skip-windows/caching/files/Triggered.scalafix.conf

```diff
@@ -0,0 +1,4 @@
+rules = [DisableSyntax]
+DisableSyntax.noVars = true
+
+  triggered.DisableSyntax.noNulls = true
```

Comment:
:+1: might be worth adding a comment just above to highlight that the whitespaces are intentional (even though the commit already says it)